app/presenters/api/v1/results_presenter.rb

```diff
@@ -24,12 +26,90 @@ def json_hash
       responses_count: responses.count,
       current_response: current_user.responses.working.find_by(time_period_id: time_period.id),
       current_user:,
-      received_and_public_shoutouts:
+      received_and_public_shoutouts:,
+      teams: teams_with_emotion_index
     }
   end
 
+  def emotion_index_all(team)
+    vars = ActiveAdminHelpers.time_period_vars(
```

Comment:
`vars` is a meaningless variable name. The fact that it's being used a lot is a bad code smell (the current approach is not great). Don't worry about now but try to think about how it can be improved sometime in the future, starting with what would calling these methods look like if it was excellent.